EIPS/eip-template.md

```diff
@@ -0,0 +1,235 @@
+---
+eip: <to be assigned>
+title: Revocation List Registry
+description: Registry of revocation lists for revoking arbitrary data.
+author: Philipp Bolte (@strumswell), Lauritz Leifermann (@lleifermann), Dennis von der Bey (@DennisVonDerBey)
+discussions-to: <URL>
```

Comment:
Please create a thread on https://ethereum-magicians.org/

Reply:
Created a post there but it needs to be unblocked by a moderator first.

Reply:
Huh, that's weird and shouldn't happen. CC @jpitts

Reply:
To be fair i never had an account there so i may have triggered some spam system by immediately posting 😄. Here's the account -> https://ethereum-magicians.org/u/lleifermann/